server/services/bookingService.ts

```diff
@@ -101,7 +101,7 @@ export default class BookingService {
         text: convertDateString(booking.expectedDepartureDate).toLocaleDateString('en-GB'),
       },
       {
-        html: `<a href="/premises/${premisesId}/bookings/${booking.id}/">
+        html: `<a href="/premises/${premisesId}/bookings/${booking.id}/departures/new">
```

Comment:
Not for now, but with this in mind, we should probably update the `Manage` links in the upcoming departures and departing today tabs to link to the departures page